### PR TITLE
starting UI Config file

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: cdap-ui/server/config/cdap-ui-config.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "tracker": {
+    "artifact": {
+      "name": "tracker",
+      "version": "1.0-SNAPSHOT",
+      "scope": "USER"
+    },
+    "appId": "_Tracker",
```
Comment (nitinmotgi) [inline cdap-ui/server/config/cdap-ui-config.json]: programId and programName - are they two different things ?

Comment (elfenheart) [inline cdap-ui/server/config/cdap-ui-config.json]: Yes, it is. Will update

**Comment by ajainarayanan** [comment]
Talked to @elfenheart We could use either Configuration JSON that we already pass from node server for CDAP or we could use a similar mechanism for passing on tracker configuration from the node server.

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by elfenheart** [comment]
demoed to @sreevatsanraman

**Pull request merged**